### ISSUE TITLE
[Accessibility] Cannot focus several buttons with Tab key

### ISSUE THREAD

**Issue by shuuji3** [jointakahe/takahe#156]
When I try to navigate the site only using a keyboard, I noticed that several buttons are skipped with the Tab key. I'm using the latest Google Chrome on macOS.

## "Add image" button
![Screenshot 2022-12-10 at 19 50 48](https://user-images.githubusercontent.com/1425259/206917051-f8271d4f-5ce5-4b71-b522-bde3f2e58fab.png)

## "CW" button
![Screenshot 2022-12-10 at 19 54 20](https://user-images.githubusercontent.com/1425259/206917054-7e50b683-a9b4-4320-83c5-7019634d7a22.png)

## menu items
<img width="276" alt="Screenshot 2022-12-12 at 1 53 31" src="https://user-images.githubusercontent.com/1425259/206917510-5149f0c1-e41b-4ac9-afca-183f42e6d21b.png">

I guess the following are the causes:
- "Add image" button is `<a>` tag with `button` class
- "CW" button is `<span>` tag with `button` class
- All menu items are `<a>` tag but has no `href` attribute except for the "Reply" menu

Probably we can change the first two elements to `<button>` to allow focusing, but I'm not sure the last case. Maybe some accessibility guide line explains how to handle a case like this.
